packages/preload/exposedInMainWorld.d.ts

```diff
@@ -1,4 +1,4 @@
 interface Window {
-    readonly fdc3: import("/Users/rob/Documents/finos/electron-fdc3/node_modules/fdc3-1.2/dist/api/DesktopAgent").DesktopAgent;
+    readonly fdc3: import("/Users/nicholaskolba/connectifi/electron-fdc3/node_modules/fdc3-1.2/dist/api/DesktopAgent").DesktopAgent;
     readonly sail: { isConnected: () => boolean; isReady: () => void; joinChannel: (channel: string) => void; leaveChannel: () => void; hideWindow: () => void; resolveIntent: (data: { selectedIntent: any; selected: { details: any; }; }) => void; versions: NodeJS.ProcessVersions; getApps: () => Promise<unknown>; tabs: { select: (selectedId: string) => void; tearOut: (tabId: string) => void; new: () => void; drop: (frameTarget: boolean) => void; dragStart: (selected: string) => void; close: (tabId: string) => void; }; menu: { openTools: (clientX: number, clientY: number) => void; openChannelPicker: (mouseX: number, mouseY: number) => void; }; search: { hideResultsWindow: () => void; searchDirectory: (query: string) => void; selectResult: (selection: string) => void; }; };
```

Comment:
this seems really weird that we are a) hard coding a path and b) linking to a specific version of the fdc3 api